src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,338 @@
+import uuid
+
+from django.core.validators import MaxValueValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.descriptors import GegevensGroepType
+
+from .constants import SoortPartij
+
+
+class Partij(models.Model):
+    id = models.UUIDField(
+        primary_key=True,
+        unique=True,
+        default=uuid.uuid4,
+        max_length=40,
+        help_text=_(
+            "Unieke (technische) identificatiecode van de betrokkene bij klantcontact."
+        ),
+    )
+    nummer = models.PositiveIntegerField(
+        _("Nummer"),
+        help_text=_(
+            "Uniek identificerend nummer dat tijdens communicatie tussen mensen kan "
+            "worden gebruikt om de specifieke partij aan te duiden."
+        ),
+        validators=[MaxValueValidator(9999999999)],
+        blank=False,
+    )
+    interne_notitie = models.CharField(
+        _("Interne notitie"),
+        help_text=_(
+            "Mededelingen, aantekeningen of bijzonderheden over de partij, bedoeld voor intern gebruik."
+        ),
+        max_length=1000,
+        blank=True,
+    )
+    soort_partij = models.CharField(
+        _("Soort partij"),
+        help_text=_("Geeft aan van welke specifieke soort partij sprake is."),
+        max_length=14,
+        choices=SoortPartij.choices,
+        blank=False,
```

Comment:
This is the default, so can be removed (...and for the other fields below, the same :sweat_smile: )

Reply:
Do you mean this for every field or just choice fields?